### DISCUSSION TITLE
Видео не находятся для скачивания

### DISCUSSION THREAD

**Discussion by Femidasky** [aclap-dev/video-downloadhelper#460] [Help and Q&A]
### Your question

Добрый день! С нужного мне сайта видео почти обнаруживаются. Приложение настроено корректно и в расширении написано, что доступ к сайту есть. Ранее скачивала с него видео без проблем, но теперь обнаружение происходит рандомно или вообще не находятся видео. Прошу помочь! 

### Addon & CoApp version and details

Video DownloadHelper 8.2.2.9 (stable)
Video DownloadHelper
Версия 8.2.2.9 (stable)
Локаль броузера: ru
Технический билд
Скомпилировано 2024-03-19
Опции компиляции: browser=chrome, linuxlic=false, noyt=true, target=google

Платформа Win x86-64
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36

**Comment by GrampaWildWilly** [comment]
> From the site I need

You would be helping yourself greatly if you would post a URL of a specific web page (or a few) on which you are experiencing problems.

**Reply by Femidasky** [reply]
https://womaninsight.com/ru/courses/ нужны видео именно с этого сайта
